app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class LikesController < ApplicationController
+  def create
+    @likeable = params[:likeable_type].constantize.find(params[:likeable_id])
+    @like = @likeable.likes.create!(user_id: current_user.id)
+    respond_to_format
+  end
+
+  def destroy
+    @like = current_user.likes.find(params[:id]).destroy
```

Comment:
same

Reply:
In this case I need it because I then use it to find the @likeable associated

Reply:
There's no need for defining it as an instance variable, you can just define it as any other variable (without the `@`)